propka/lib.py

```diff
@@ -241,8 +243,7 @@ def build_parser(parser=None):
         help=("specifying mutation labels which is used to modify "
               "<filename> according to, e.g. N25R/N181D"))
     group.add_argument(
-        "-v", "--version", dest="version_label", default="Jan15",
-        help="specifying the sub-version of propka [Jan15/Dec19]")
+        "--version", action="version", version=f"%(prog)s {propka.__version__}")
```

Comment:
Is there a "print" action anywhere that prints the version and exits?

Reply:
If there's a separate print_version() function then one could have the `import propka` confined to this small function and ensure that it only happens when the version _is_ printed (and then the code already exists).

Or does `--version` work automagically?

Reply:
`version` is one of the available actions from `argparse`, so this works automatically.
https://docs.python.org/3/library/argparse.html#action